util/java/libtiled-java/src/main/java/tiled/core/Map.java

```diff
@@ -304,8 +402,19 @@ public String getFilename() {
      * Returns a vector with the currently loaded tileSets.
      *
      * @return Vector
+     * @deprecated Use getMapTileSets instead.
```

Comment:
Do we really need to do this? So far there aren't any compatibility guarantees and it should be a small thing to fix for projects upgrading to a new libtiled-java if we just change the return type, right?